happo_example.sh

```diff
@@ -16,7 +16,7 @@ run-happo() {
   xvfb-run happo
 }
 
-# Check out the previous version and generate baseline snapshots
+# Check out the previous version and generate previous snapshots
```

Comment:
This sentence is a little strange. Maybe you can just leave out previous from the second part:

"Check out the previous version and generate snapshots"